src/controller/index.ts

```diff
@@ -412,6 +419,24 @@ export default class SimulariumController {
         return this.playBackFile;
     }
 
+    public checkServerHealth(
+        handler: () => void,
+        netConnectionConfig: NetConnectionParams
+    ): void {
+        if (
+            !this.simulator ||
+            !(this.simulator instanceof RemoteSimulator) ||
+            !this.simulator.socketIsValid()
+        ) {
+            // Only configure network if we aren't already connected to the remote server
+            this.configureNetwork(netConnectionConfig);
+        }
+        if (this.simulator && this.simulator instanceof RemoteSimulator) {
+            this.simulator.setHealthCheckHandler(handler);
+            this.simulator.checkServerHealth();
```

Comment:
should checkServerHealth accept a callback as an arg instead of having a separate setter?  Are they always called together like this?

Reply:
I see why they are separate now, because of the message passing and receiving.